src/mon/PGMap.cc

```diff
@@ -824,8 +824,9 @@ void PGMapDigest::dump_object_stat_sum(
     curr_object_copies_rate = (float)(sum.num_object_copies - sum.num_objects_degraded) / sum.num_object_copies;
 
   float used = 0.0;
+  // note avail passed in is raw_avail, calc raw_used here.
   if (avail) {
-    used = sum.num_bytes * curr_object_copies_rate;
+    used = sum.num_bytes * raw_used_rate * curr_object_copies_rate;
```

Comment:
I know this is a backport, but I don't understand *at all* why we are multiplying by both the raw_used_rate (which is ```pool->get_size()``` for replicated pools) and the curr_object_copies_rate.

...oh, it's because curr_object_copies_rate is actually about how many objects *actually exist* compared to those that are *supposed to exist*.